### ISSUE TITLE
Only 1 cpu usgae

### ISSUE THREAD

**Issue by Alimassali** [stotiks/chia-plotter#9]
Will only run 1 cpu at 100% in a 2 cpu setup. Is there anything I can do? Or do we need an update?

Thanks

**Comment by XxViper06xX** [comment]
> Will only run 1 cpu at 100% in a 2 cpu setup. Is there anything I can do? Or do we need an update?
> 
> Thanks

Seems to be working fine for me. Dual Intel E5-2699v3. I have an issue with it hanging up sometimes but CTRL+C gets it going again.

**Comment by Alimassali** [comment]
> > Will only run 1 cpu at 100% in a 2 cpu setup. Is there anything I can do? Or do we need an update?
> > Thanks
> 
> Seems to be working fine for me. Dual Intel E5-2699v3. I have an issue with it hanging up sometimes but CTRL+C gets it going again.

Hmm wondering why mine is not working.
Did try WSL Ubuntu as well but capped at 1 cpu :(

**Comment by XxViper06xX** [comment]
> > > Will only run 1 cpu at 100% in a 2 cpu setup. Is there anything I can do? Or do we need an update?
> > > Thanks
> > 
> > 
> > Seems to be working fine for me. Dual Intel E5-2699v3. I have an issue with it hanging up sometimes but CTRL+C gets it going again.
> 
> Hmm wondering why mine is not working.
> Did try WSL Ubuntu as well but capped at 1 cpu :(

Are you running Alpha v0.0.1 or 0.0.3? I'm not having this issue in v0.0.3

**Comment by XxViper06xX** [comment]
> > > > Will only run 1 cpu at 100% in a 2 cpu setup. Is there anything I can do? Or do we need an update?
> > > > Thanks
> > > 
> > > 
> > > Seems to be working fine for me. Dual Intel E5-2699v3. I have an issue with it hanging up sometimes but CTRL+C gets it going again.
> > 
> > 
> > Hmm wondering why mine is not working.
> > Did try WSL Ubuntu as well but capped at 1 cpu :(
> 
> Are you running Alpha v0.0.1 or 0.0.3? I'm not having this issue in v0.0.3 Also I'm running in Windows.

Sorry but I just noticed it only using one CPU as well. I had other Processes using other CPU causing me to think it was using both.
![Mad Max only 1 CPU](https://user-images.githubusercontent.com/85854828/121830749-57c54a80-cc8b-11eb-8948-4108273ab3f4.jpg)

**Comment by XxViper06xX** [comment]
Confirmed. Flagged to use 32 here.

![Mad Max only 1 CPU(1)](https://user-images.githubusercontent.com/85854828/121831050-2436f000-cc8c-11eb-86db-db1d7e346a1e.jpg)

**Comment by jrharp** [comment]
I don't have any screen shots, but I have had the same issue with a single processor (Xeon Gold 6226r), it has 16 cores and 32 threads, but it only ever seems to utilize 16 threads.  I thought it was related to not using the hyper threading, my task manager looked very similar to what is being posted here.

**Comment by Alimassali** [comment]
> > > > Will only run 1 cpu at 100% in a 2 cpu setup. Is there anything I can do? Or do we need an update?
> > > > Thanks
> > > 
> > > 
> > > Seems to be working fine for me. Dual Intel E5-2699v3. I have an issue with it hanging up sometimes but CTRL+C gets it going again.
> > 
> > 
> > Hmm wondering why mine is not working.
> > Did try WSL Ubuntu as well but capped at 1 cpu :(
> 
> Are you running Alpha v0.0.1 or 0.0.3? I'm not having this issue in v0.0.3 Also I'm running in Windows.

Yea latest version as well

**Comment by Alimassali** [comment]
> > > > Will only run 1 cpu at 100% in a 2 cpu setup. Is there anything I can do? Or do we need an update?
> > > > Thanks
> > > 
> > > 
> > > Seems to be working fine for me. Dual Intel E5-2699v3. I have an issue with it hanging up sometimes but CTRL+C gets it going again.
> > 
> > 
> > Hmm wondering why mine is not working.
> > Did try WSL Ubuntu as well but capped at 1 cpu :(
> 
> Are you running Alpha v0.0.1 or 0.0.3? I'm not having this issue in v0.0.3 Also I'm running in Windows.

Yes I'm on latest version, but i think this might me a windows issue ?!

**Comment by Alimassali** [comment]
I thought this was a madmax problem but i just realized that even on swar plot manager only 1 cpu was at 100% ,showing only 60% usage which means 1 I'd at 100% and the rest is being used from cpu 2 for windows chrome etc.

**Comment by XxViper06xX** [comment]
> I thought this was a madmax problem but i just realized that even on swar plot manager only 1 cpu was at 100% ,showing only 60% usage which means 1 I'd at 100% and the rest is being used from cpu 2 for windows chrome etc.

I'll have to look into this as well. I haven't paid much attention to Swar. I was getting the same results as the person I copied when I ran it, so that never crossed my mind.
